scripts/ts-utils/setup-enclave.ts

```diff
@@ -66,20 +95,21 @@ async function updateScheduledEnclave(api: any, signAccount: any) {
 
 async function main() {
 
+    // NOTE: If we are using development environment, we can use Alice instead of moving around with keys
```

Comment:
Do you expect this enclave update script to only work for dev, or is it suitable for prod/staging too?

Reply:
So here there are two things, It is considered to work for any environment that is using the `dev` chain because here the `sudo` is set to be `Alice`. This is not suitable for Mainnet wherein I guess we shouldn't be calling `sudo` calls via script because then we would have to store the private key on the server running this script. If we consider staging to be a network using `dev` then this should be alright.

Reply:
The staging's root isn't Alice though 😂 (otherwise anyone could do anything on tee-staging parachain)